doc/user/content/installation/install-on-aws/_index.md

```diff
@@ -73,27 +73,10 @@ modules](https://github.com/MaterializeInc/terraform-aws-materialize/blob/main/R
 for evaluation purposes only. The modules deploy a sample infrastructure on AWS
 (region `us-east-1`) with the following components:
 
-- A Kuberneted (EKS) cluster
-- A dedicated VPC
-- An S3 for blob storage
-- An RDS PostgreSQL cluster and database for metadata storage
-- Materialize Operator
-- Materialize instances (during subsequent runs after the Operator is running)
-
-- **Starting in v0.3.0 of Materialize on AWS Terraform**, AWS Load Balancer
-  Controller and AWS Network Load Balancers (NLBs) for each
-  Materialize instance. If your deployment is set up using an earlier version of
-  the Materialize AWS Terraform module, and you wish to upgrade to v0.3.0
-  Terraform modules, see the [Upgrade
-  Notes](https://github.com/MaterializeInc/terraform-aws-materialize/blob/main/README.md#upgrade-notes).
-
-- **Starting in v0.3.1 of Materialize on AWS Terraform**, OpenEBS and NVMe
-  instance storage to [enable
-  spill-to-disk](https://github.com/MaterializeInc/terraform-aws-materialize?tab=readme-ov-file#enabling-disk-support)
-  to support workloads that are larger than can fit into memory.
+{{< yaml-table data="self_managed/aws_terraform_deployed_components" >}}
```

Comment:
https://preview.materialize.com/materialize/32065/self-managed/v25.1/installation/install-on-aws/#set-up-aws-kubernetes-environment-and-install-materialize